job/scriptsGen/normalizeSteps.js

```diff
@@ -66,7 +66,7 @@ function _normalizeNewFormatSteps(steps, defaultRuntime, onSuccess,
   // Default options for container tasks.
   var defaultContainerOpts = {
     imageName: imageName,
-    imageTag: global.config.shippableReleaseVersion,
```

Comment:
This was running on the current release version to keep the behavior same as genExec (ie. run on latest microbase) until we had a resolution on https://github.com/Shippable/heap/issues/1975. Since we don't have `prod` tag for stable/unstable microbase and potentially something might have changed in other microbase images until v6.3.2, this is a risky change.

Reply:
I think we have three options:
- Retag all the missing/existing microbase to the last known microbase.
- Add a conditional check in the code to only use the drydockTag if the version is greater than the last microbase version.
- Add a column in runtimeTemplates exclusively for taskContainerImageTag.

Reply:
hmm, this was supposed to be the resolution on 1975, but none of those details were mentioned in the original heap. 

how about:
if defaultTaskImage is "microbase", then set a certain tag. in the future, all defaultTaskImages hsould be `u16microbase` or `u14microbase`, etc, there shouldnt be anymore microbase, so that should preserve today's runtimeTemplates while allowing future ones to follow new pattern

Reply:
if your first point you mean retag in the templates, problem with changing the templates is that CI uses them too, and we dont want CI to use 6.2.4 for everything, but in this case we do want runSh to use 6.2.4 for everything.

Reply:
i think conditional check is the way to go since we can basically say any taskImage of "microbase" should have tag "v6.2.4"... i'll add this...

Reply:
perhaps column is the better long term fix, but to unblock i think conditional check is best

Reply:
Should work. The only thing is there might be some confusion about how we pick the default image as the older `runtimeTemplates` will run on the latest version inspite of picking an older version (which exists even today, but the behavior is same across all versions, which will now change). So, I guess if we want to go with this, we should maybe show what the default task container will be in the UI later on, so that there is no confusion.

Reply:
Actually, we will not be able to change `microbase` to have update packages. So, that downside wil still remain, unless we have no plans of updating it.

Reply:
yeah, i assumed that with this new way we would just tell customers to update their template to get the new features.

but yeah, it will be a little confusing as everything 6.2.4 and < will use 6.2.4, and everything after will be based on the selected template.

Reply:
ahh... hmm... actually the retagging doesn't seem like a bad idea to me :) could be done quickly with no code change... then this change could stay... though i guess we'd be overriding old microbase images... but they arent used for anything right?

Reply:
Yeah, I don't think it will have any downsides as all the dependent images have already been built and no one should be using microbase directly for anything as its "internal" and was only recently exposed in reqProc.